cooperator/models/partner.py

```diff
@@ -85,6 +85,8 @@ def _compute_effective_date(self):
         for partner in self:
             if partner.share_ids:
                 partner.effective_date = partner.share_ids[0].effective_date
+            else:
+                partner.effective_date = False
```

Comment:
because [compute functions must always set a value](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration) (in 13.0, but like this we keep differences between 12.0 and 14.0 to a minimum).